### DISCUSSION TITLE
Can I use a joystick?

### DISCUSSION THREAD

**Discussion by shun9092** [cathery/sys-con#46] [Q&A]
Can I use a joystick?

I tried to connect a Razer Panthera(not evo) but it doesn't work.

**Comment by cathery** [comment]
You might, in the future. DInput/PS4 controllers don't work yet. If you can set your joystick to XInput it should work.

**Comment by shun9092** [comment]
I switched to Dual Shock 3 mode and D-pad, than connected.

But it didn't work.

**Comment by oDHAOSo** [comment]
Sys-con is working for me with a DS3, but does not work with my MadKatz fightstick pro for ps3, or my stick using a "BrookAccessory Universal Fighting Board" PCB

**Comment by cathery** [comment]
DInput is not supported yet.
